test/JDBC/input/BABEL_COL_NAME-vu-prepare.sql

```diff
@@ -0,0 +1,66 @@
+CREATE DATABASE babel_3172_test_db;
+GO
+
+USE babel_3172_test_db;
+GO
+
+CREATE TABLE sys_column_name_vu_t_column_name(
+    id int,
+    names char
+)
+GO
+
+CREATE SCHEMA sys_col_name_test_schema;
+GO
+
+CREATE TABLE sys_col_name_test_schema.test_table(
+    firstName varchar(30)
+)
+GO
+
+CREATE VIEW col_name_prepare_v1 AS (SELECT COL_NAME(CAST((SELECT OBJECT_ID('sys_column_name_vu_t_column_name')) AS INT), 1))
+GO
+
```

Comment:
Can you please add testcase with hexadecimal, decimal , text values as input . And also try using NULL on both input parameter if not done already.